src/I18n/DateFormatTrait.php

```diff
@@ -64,9 +74,57 @@
     protected static $_isDateInstance;
 
     /**
+     * Gets the default output timezone.
+     *
+     * @return \DateTimeZone|null DateTimeZone object in which the date will be displayed or null.
+     */
+    public static function getDefaultOutputTimezone()
+    {
+        return static::$_defaultOutputTimezone;
+    }
+
+    /**
+     * Sets the default output timezone.
+     *
+     * @param string|\DateTimeZone $timezone Timezone string or DateTimeZone object
+     * in which the date will be displayed.
+     * @return void
+     */
+    public static function setDefaultOutputTimezone($timezone)
+    {
+        if ($timezone === null) {
+            $timezone = new \DateTimeZone(date_default_timezone_get());
+        } elseif (is_string($timezone)) {
+            $timezone = new \DateTimeZone($timezone);
+        }
+        static::$_defaultOutputTimezone = $timezone;
```

Comment:
Should probably check that this is a DateTimeZone before setting the property, as we rely on it being a timezone later on.